v2/apiserver/internal/core/events.go

```diff
@@ -663,6 +666,17 @@ func (e *eventsService) DeleteMany(
 						event.ID,
 					))
 				}
+
+				if err := e.logsStore.DeleteEventLogs(
+					context.Background(), // deliberately not using ctx
+					event.ID,
+				); err != nil {
+					log.Println(errors.Wrapf(
+						err,
+						"error deleting logs for event %q",
+						event.ID,
+					))
+				}
```

Comment:
I love that this is being done in the same fanned-out goroutines we're using for cleaning up the substrate.